src/regedit/register.jl

```diff
@@ -209,7 +212,7 @@ function register(
         vslist = [(string(k), v) for (k, v) in vnlist]
 
         open(versions_file, "w") do io
-            TOML.print(io, OrderedDict(vslist))
```

Comment:
The `OrderedDict` was for sorting the versions https://github.com/JuliaComputing/Registrator.jl/issues/11

Are you sure this is not needed anymore?

Reply:
Good catch, but we can just use
```
TOML.print(io, vslist; sorted=true, by = VersionNumber)
```
instead of an `OrderedDict` since `TOML.print` has such a kwarg.

Reply:
Cool, thanks!